### DISCUSSION TITLE
Throw errors when present from sdk

### DISCUSSION THREAD

**Discussion by GuyShane** [directus/directus#22670] [Feature Requests]
### Describe the Improvement

When I'm using the sdk with graphql, and there is any sort of error during a request, it seems to get swallowed y the sdk and there's no way to access it on the client side. 
I'm making some validation extensions for my app, and when they return errors I'd like to be able to surface them to the user, but there doesn't currently seem to be a way to do that.
I'm proposing that if there is an `errors` field present in a JSON response, the sdk should throw.

**Comment by br41nslug** [comment]
oops sorry for the ping pong between discussion and issue, i had misread the issue 😅 https://github.com/directus/directus/issues/22671
